chapters/synchronous.tex

```diff
@@ -443,7 +443,7 @@ \section{Clock Constructors}\label{clock-constructors}
 \end{operatordefinition*}
 
 Besides inferred clocks and solver clocks, one of the following mutually
-exclusive associations of clocks are possible in one base partition:
+exclusive associations of clocks are possible in one base-clock partition:
```

Comment:
Time to go sentence-based, but for some reason I can't make a Github suggestion.